ci/debian.cross-compile.sh

```diff
@@ -22,6 +22,9 @@ apt install -y --no-install-recommends \
 	libftdi-dev:${ARCH} \
 	libudev-dev:${ARCH} \
 	libyaml-dev:${ARCH} \
+	pkg-config:${ARCH} \
+	meson \
+	ninja-build \
```

Comment:
You don't have to repeat this, they should be installed already by `ci/debian.sh`.